.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,43 @@
+# Learn more about CodeQL at https://codeql.github.com/docs/
```

Comment:
After talking with @diegopf we believe current issues aren't a real problem for now:

- `interpolate` is used to craft endpoint urls. We control the template they work with.
  - If we support SSR at some point this might become a potential problem, but as long as we control the input data it shouldn't be real.
- `cleanMarkdown` is used as part of the build process. The parameter passed is  part of our source code.
- Prototype pollution in the filters/scroll shouldn't be an issue either as we own that data.

Reply:
So, regarding this, could we [exclude](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#avoiding-unnecessary-scans-of-pull-requests) these files from being scanned? And, maybe for the `interpolate` function, we could add a `TODO` for including it once we support SSR.

Reply:
I would rather dismiss the alert than exclude the file. For the `interpolate` issues we can mark them as won't fix and the `clearMarkdown` we can say it is not part of the build.